### PR TITLE
Auditd: Add documentation for the role

### PR DESCRIPTION
**Pull request by ra-rau**
Part of osism/issues#185

Signed-off-by: Ramona Rautenberg <rautenberg@osism.tech>

### CODE REVIEW

File: roles/auditd/README.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
+Ansible Role for installation and configuration Auditd.
+Auditd is used for collecting Package security messages and forwarding
+them to a central server.
+
+**Role Variables**
+
+.. zuul:rolevar:: auditd_backup
+   :default: false
+
+Sets the backup to false. If true the role will make a backup from the
```
Comment (osfrickler) [inline roles/auditd/README.rst:10]: Drop the first sentence.

File: roles/auditd/README.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+rule files. 
+
+.. zuul:rolevar:: auditd_syslog
+   :default: false
+
+This varaible disables write to rsyslog for auditd.
+
+
+**Variable for Configuration**
```
Comment (osfrickler) [inline roles/auditd/README.rst:19]: Drop these additional headers, they are confusing

File: roles/auditd/README.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+.. zuul:rolevar:: auditd_config
+
+.. code-block:: yaml
+
+   - parameter: active
+     value: "{{ 'yes' if auditd_syslog|bool else 'no' }}"
+     config: /etc/audisp/plugins.d/syslog.conf
+
+Configuration for Auditd.
```
Comment (osfrickler) [inline roles/auditd/README.rst:29]: A list of configuration values to set for auditd. Each entry is a hash with three items:

parameter: The name of the parameter to set
value: The value the parameter should be set to
config: The name of the configuration file in which the parameter is set

File: roles/auditd/README.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+
+**Variables for Rules**
+
+.. zuul:rolevar:: auditd_rules_path
+   :default: /etc/audit/rules.d
+
+Path to where the rules from Auditd should be stored.
```
Comment (osfrickler) [inline roles/auditd/README.rst:37]: Path where the rules should be stored.

File: roles/auditd/README.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+The configuration files for the rules of Auditd composed from the Variables
+auditd_rules_files_defaults and auditd_rules_files_extra.
+
+
+**Variables for Packages**
+
+.. zuul:rolevar:: auditd_package_name
+   :default: auditd
+
+The required package for Auditd.
```
Comment (osfrickler) [inline roles/auditd/README.rst:60]: The name of the auditd package

File: roles/auditd/README.rst [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+.. zuul:rolevar:: audispd_plugins_package_name
+   :default: audispd-plugins
+
+Plugins for Auditd.
```
Comment (osfrickler) [inline roles/auditd/README.rst:65]: The name of the audispd-plugins package.

File: roles/auditd/README.rst [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+
+**Variable for Services**
+
+.. zuul:rolevar:: auditd_service_name
+   :default: auditd
+
+Service name to deal with the Auditd service.
```
Comment (osfrickler) [inline roles/auditd/README.rst:73]: The name of the auditd service

**Comment by berendt** [comment]
Merging as is. We will cleanup everything in one followup PR.

**Pull request merged**